srcpkgs/NetworkManager-fortisslvpn/template

```diff
@@ -0,0 +1,37 @@
+# Template file for 'NetworkManager-openvpn'
```

Comment:
This should be the same as the package name.

Reply:
I’m surprised I didn’t catch that earlier. I’ll change that now.